lib/matterhorn/links/relation/base.rb

```diff
@@ -124,9 +124,11 @@ def find_with_ids(resource, ids)
           scope(resource).in(inverse_field_key => ids)
         end
 
-        def get_items_ids(items)
+        def get_items_ids(resource, items)
+          resource = [resource].flatten.inject({}){ |hsh, object| hsh[object[:_id]] = object; hsh}
```

Comment:
Merging this now would load the whole scope multiple times from the database (specifically one for each inclusion).  It should be memoized.

Can you add a spec asserting that we only load from db once?